Example/Podfile

```diff
@@ -1,6 +1,14 @@
 platform :ios, '13.0'
 
+plugin 'cocoapods-bazel', {
+  rules: {
+    'apple_framework' => { load: '@build_bazel_rules_ios//rules:framework.bzl', rule: 'apple_framework' }.freeze,
+    'ios_application' => { load: '@build_bazel_rules_ios//rules:app.bzl', rule: 'ios_application' }.freeze,
+    'ios_unit_test' => { load: '@build_bazel_rules_ios//rules:test.bzl', rule: 'ios_unit_test' }.freeze
+  }.freeze,
+}
+
 target 'FLEXample' do
   project 'FLEXample-Cocoapods'
-  pod 'FLEX', :path => '../'
+  pod 'FLEX', :git => 'https://github.com/gojekfarm/Flex.git', :branch => 'add-header-maps', :modular_headers => true , :configurations => ['Debug']
```

Comment:
To ensure backward compatibility and no changes to existing users.

Let's merge this change into a new branch called `remove-private-api-usage-and-add-header-maps`

Because at the moment, anyone who runs pod repo update or pod install after deleting podfile.lock will get these changes included, which may cause issues for some of other apps.

Let me know what you think. If you feel this change should be harmless, we can also merge it into the existing `remove_private_api_usage` branch.

Reply:
Yeah let's keep this in a separate branch that way it would be easy to revert also if something goes wrong. Ok keeping these changes in `remove-private-api-usage-and-add-header-maps`